src/components/Navbar.js

```diff
@@ -8,13 +9,13 @@ const Navbar = () => {
     return null;
   } else
     return (
-      <nav>
-        <ul className="links">
+      <nav className="bg-yellow-200 fixed w-full bottom-0">
+        <ul className="flex justify-center gap-32 py-1 ">
           <li>
-            <NavLink to="/list">List</NavLink>
+            <NavLink to="/list" className="flex flex-col items-center"><TiShoppingCart className="text-4xl" /><span>List</span></NavLink>
           </li>
           <li>
-            <NavLink to="/add-item">add an item</NavLink>
+            <NavLink to="/add-item" className="flex flex-col items-center"><TiPlusOutline className="text-4xl" /><span>Add Item</span></NavLink>
```

Comment:
Tried to use active class here rather than in css file, but I couldn't get it to stay after being clicked.

Reply:
The `active` class is generated by `<NavLink />` so I don't think we can do with tailwind. it's fine to stay in the CSS file I guess.

Reply:
That´s right. Using the CSS file makes sense in this case. Wonderful job Debbie! It´s great that you included before and after images 😄

Reply:
Thank you both.